README.md

```diff
@@ -53,7 +53,7 @@ You can use any file format, but here is a list of common media types for foreca
 | Type                    | Description |
 | ----------------------- | ----------- |
 | `application/wmo-GRIB2` | GRIB2 file (usually with the file extension `.grb2` or `.grib2`) |
-| `application/netcdf`    | NetCDF file (usually with the file extension `.nc`) |
+| `application/x-netcdf`    | NetCDF file (usually with the file extension `.nc`) |
```

Comment:
I've seen both being used. Do you have a source of which is recommended to use by the authors of the spec or so?

Reply:
I could not find it in IANA: https://www.iana.org/assignments/media-types/media-types.xhtml

"The x- prefix of a MIME subtype-identifier implies that it is non-standard i.e. not registered with IANA" - https://www.nih-cfde.org/resource/intro-mime-type/

Reply:
```suggestion
| `application/x-netcdf`  | NetCDF file (usually with the file extension `.nc`) |
```